examples/beam/src/test/java/edu/snu/nemo/examples/beam/AlternatingLeastSquareITCase.java

```diff
@@ -33,19 +35,32 @@
   private static final int TIMEOUT = 180000;
   private static final String fileBasePath = System.getProperty("user.dir") + "/../resources/";
   private static final String input = fileBasePath + "sample_input_als";
+  private static final String outputFileName = "sample_output_als";
+  private static final String output = fileBasePath + outputFileName;
+  private static final String testResourceFileName = "test_output_als";
   private static final String numFeatures = "10";
   private static final String numIteration = "3";
+  // If you don't want to check validity of output, make this variable false.
+  private static final boolean checkOutput = true;
 
   private static ArgBuilder builder = new ArgBuilder()
```

Comment:
I think we can simply declare the variable here, as `@Before` is called before every `@Test`, and `setUp()` will assign the variable with a value.

Something like
```java
private ArgBuilder builder; 
```